### PR TITLE
database cleanup

### PR DESCRIPTION
**Pull request by wkg7vb**

### CODE REVIEW

File: BAR/Data/Models/ApplicationUser.cs [hunk 1]
```diff
@@ -9,10 +9,44 @@
...
+        //id
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public required string GUID { get; set; }
+        public required string UserName { get; set; }
+        public required string Password { get; set; }
+
+        //profile
+        [AllowNull]
+        public byte[] UserImage { get; set; }
```
Comment (max-dejesus) [inline BAR/Data/Models/ApplicationUser.cs]: for this i was looking at no images for the user, but they can pick a profile icon color. thoughts?

File: BAR/Data/Models/ApplicationUser.cs [hunk 2]
```diff
@@ -9,10 +9,44 @@
...
+
+        [AllowNull]
+        public string FName { get; set; }
+
+        [AllowNull]
+        public string LName { get; set; }
+
+        //settings
+        public bool Darkmode { get; set; }
+        public char Locale { get; set; }
```
Comment (max-dejesus) [inline BAR/Data/Models/ApplicationUser.cs]: locale is more than one character so maybe str?
ex. en-US, es-MX

File: BAR/Data/Models/ApplicationUser.cs [hunk 3]
```diff
@@ -9,10 +9,44 @@
...
+        public required string GUID { get; set; }
+        public required string UserName { get; set; }
+        public required string Password { get; set; }
+
+        //profile
+        [AllowNull]
+        public byte[] UserImage { get; set; }
+
+        [AllowNull]
+        public string FName { get; set; }
```
Comment (max-dejesus) [inline BAR/Data/Models/ApplicationUser.cs]: may want to require f/l name

**Review by max-dejesus — CHANGES_REQUESTED** [review]

File: BAR/Data/Models/ApplicationUser.cs (hunk 1 above)
Comment (wkg7vb) [reply BAR/Data/Models/ApplicationUser.cs]: i think will be just fine, discuss more during friday meeting

File: BAR/Data/Models/ApplicationUser.cs (hunk 2 above)
Comment (wkg7vb) [reply BAR/Data/Models/ApplicationUser.cs]: done

File: BAR/Data/Models/ApplicationUser.cs (hunk 3 above)
Comment (wkg7vb) [reply BAR/Data/Models/ApplicationUser.cs]: done

**Review by max-dejesus — APPROVED** [review]
thanks

File: BAR/Data/Models/ApplicationUser.cs [hunk 4]
```diff
@@ -9,10 +9,44 @@
 namespace BAR.Data.Models
 {
     // Add profile data for application users by adding properties to the ApplicationUser class
-    public class ApplicationUser : IdentityUser
```
Comment (wkg7vb) [inline BAR/Data/Models/ApplicationUser.cs:12]: removing " : IdentityUser" is what broke main.

**Pull request merged**